frontend/mulighetsrommet-veileder-flate/src/views/tiltaksgjennomforing-detaljer/ViewTiltaksgjennomforingDetaljer.tsx

```diff
@@ -138,7 +138,10 @@ const ViewTiltaksgjennomforingDetaljer = ({
       <div className={styles.container}>
         <div className={styles.top_wrapper}>
           {!erPreview && (
-            <Tilbakeknapp tilbakelenke={`/oversikt/#page=${page}`} tekst="Tilbake til tiltaksoversikten" />
+            <Tilbakeknapp
+              tilbakelenke={`/arbeidsmarkedstiltak/oversikt/#page=${page}`}
```

Comment:
Litt sidespor dette, men når vi først er inne på routing: kunne ikke gjort om hash-style paremeters til standard path params? :grin: 
Altså path `/arbeidsmarkedstiltak/oversikt/:page`.

Reply:
Jo, godt poeng. Er faktisk litt usikker på om vi faktisk bruker page fra url'en nå, eller om alt ligger i jotai-filteret. Jeg sjekker.

Reply:
Dersom vi skal ha page i url'en så må vi også ha page i url'en inne på detaljsiden for å huske hvilken side du sto på når du går tilbake.